x-pack/plugins/cloud_security_posture/kibana.json

```diff
@@ -10,6 +10,6 @@
   "description": "The cloud security posture plugin",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["navigation", "data", "fleet"],
+  "requiredPlugins": ["navigation", "data", "fleet", "unifiedSearch"],
```

Comment:
On behalf of the Cloud Security Posture team:
We're also importing `dataPluginMock` from `src/plugins/data/public/mocks` in `x-pack/plugins/cloud_security_posture/public/test/test_provider.tsx:13` - should the import change there as well?

Reply:
No the mocks will stay on the data plugin. We just moved the Search bar logic (the frontend component) on its own plugin.